src/frontend/js/reducers/security.js

```diff
@@ -1,23 +1,41 @@
 import {
   CHECK_LOGGED_IN,
+  LOGIN_REQUESTED,
+  LOGIN_SUCCSSFUL,
+  LOGIN_FAILED,
 } from '../actions/Security';
 
 function checkLoggedIn() {
   const token = localStorage.getItem('token');
   if (token == null) {
     return { loggedIn: false, family: null };
   }
+  return userLoggedIn(token);
+}
+
+function userLoggedIn(token) {
+  localStorage.setItem('token', token);
   const data = JSON.parse(atob(token.split('.')[1]));
```

Comment:
I'm not sure if this variable is adding anything? Could just do `family: JSON.parse(atob(token.split('.')[1])).sub` or change the name of `data` to something with more meaning?

Reply:
Agree, removing it.